bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/ndk/NativeBridge.kt

```diff
@@ -120,6 +123,9 @@ class NativeBridge : StateObserver {
                 updateUserEmail(makeSafe(event.user.email ?: ""))
             }
             is StateEvent.UpdateMemoryTrimEvent -> updateLowMemory(event.isLowMemory, event.memoryTrimLevelDescription)
+            is StateEvent.AddFeatureFlag -> addFeatureFlag(event.name, event.variant)
+            is StateEvent.ClearFeatureFlag -> clearFeatureFlag(event.name)
```

Comment:
calls in the nativebridge needs to go through `makeSafe()` (which forces the encoding to UTF-8)